core/src/main/java/com/graphhopper/routing/util/CarFlagEncoder.java

```diff
@@ -186,6 +187,14 @@ protected double getSpeed(ReaderWay way) {
         return speed;
     }
 
+    @Override
+    public boolean acceptsTurnRelation(OSMTurnRelation relation) {
+        if (relation.isVehicleTypeConcernedByTurnRestriction(restrictions)) {
```

Comment:
just `return relation.isVehicleTypeConcernedByTurnRestriction(restrictions)` (here and in other encoders)

Reply:
ok